torch/_dynamo/variables/builder.py

```diff
@@ -108,6 +108,7 @@ class GraphArg:
     source: Source
```

Comment:
Is this guaranteed to be a Tensor? What about dynamic int arguments?

Reply:
Added checks